standard/clause_specification_text.adoc

```diff
@@ -15,6 +15,9 @@ The following is an illustrative example of using CoverageJSON to encode air tem
 [%unnumbered%]
 ```json
 {
+  "conformsTo": [
+    "http://www.opengis.net/spec/covjson/1.0"
```

Comment:
"http://www.opengis.net/spec/covjson/1.0.1"

Reply:
Agreed bug fix level needed in vesion in case of errors in schemas.

Reply:
If we choose to emit "x.y.z" as part of the conformance URI, then all bug fix releases will require conformance URI updates in all downstream applications.  This is inconsistent with how OGC API standards, for example, identify conformance (x.y, even for bugfix releases).  Is this the desired behaviour?

Reply:
@tomkralidis Aah! When was the decision to have requirements and conformance downstream of version the schemas? I was thinking landing pages:
 http://www.example.com/oapi-whatever/collections?...
 http://www.example.com/oapi-whatever/conformance?...
 http://www.example.com/oapi-whatever/api?...

and thinking of conformance at the spec level rather than every single conformance class level? ( Is OGC making the APIs too detailed/fine grained? )